draft-irtf-cfrg-vdaf.md

```diff
@@ -1643,6 +1658,12 @@ def agg_shares_to_result(Prio3, _agg_param, agg_shares,
 ### Auxiliary Functions {#prio3-helper-functions}
 
 ~~~
+def joint_rand(Prio3, k_joint_rand_parts):
```

Comment:
This isn't really an issue with this PR, but is there a reason these function declarations can't have type annotations on their arguments, like we do for `derive_seed` or `expand_into_vec` in the PRG section? It would be a big help to readers trying to understand if `k_joint_rand_parts` is a `Bytes` (all parts already concatenated) or a `[Bytes]` (each element is a part) or some other thing.

Reply:
There's no rhyme or reason for this. I'm happy to add type hints here.